pkg/util/image_test.go

```diff
@@ -50,3 +50,69 @@ func TestGetRepoCreds(t *testing.T) {
 		assert.Equal(t, tc.p, pass, msg)
 	}
 }
+
+func TestParseImageSpec(t *testing.T) {
+	cases := []struct {
+		name      string
+		imageStr  string
+		server    string
+		repo      string
+		shouldErr bool
```

Comment:
util.ParseImageSpec will never return an error. This isn't needed.

I would remove the error return value from util.ParseImageSpec.